### ISSUE TITLE
Error connecting to Azure subscription with ML extension - reason: unable to get issuer certificate

### ISSUE THREAD

**Issue by dj674** [microsoft/vscode-tools-for-ai#2472]
## Expected Behavior
Having installed the Machine Learning extension on the Virtual desktop and selecting "Sign In to Azure", User will be redirected to a webpage to select their user account, and is then able to sign in and see subscriptions and machine learning workspaces/instances to connect to.

## Actual Behavior
When trying to "Sign In to Azure", User will be redirected to a webpage to select their user account, user selects their user account then gets the error:-
2024-11-21 11:08:05.780 [info] [email https://management.azure.com/.default offline_access openid profile] Creating session
2024-11-21 11:09:22.591 [error] [email https://management.azure.com/.default offline_access openid profile] Fetching token failed: request to https://login.microsoftonline.com/organizations/oauth2/v2.0/token failed, reason: unable to get issuer certificate
2024-11-21 11:09:22.591 [error] [email https://management.azure.com/.default offline_access openid profile] Error exchanging code for token: Error: Network failure
2024-11-21 11:09:22.595 [info] [email https://management.azure.com/.default offline_access openid profile] Getting sessions

## Steps to Reproduce the Problem
  1. Install machine learning extension.
  2. Select  "Sign In to Azure"
  3. Select User account

## Specifications

Product Vscode:
  - Version: 1.94.2
  - Platform: Windows_NT x64

**Comment by JerryYangKai** [comment]
Hello @dj674,  sorry for the delay.

Does this issue still exist?  
I try your repro step but cannot reproduce it, can you provide more information about it?

**Comment by dj674** [comment]
This is now fixed. We needed to change Microsoft authentication from classic to msal and it connected.
